### PR TITLE
LEAF-1937 - Save button float issue for Workflow Add Event popup

### PR DESCRIPTION
**Pull request by gsteinhoff**
Fix was to remove un-needed floats causing confusion in IE

### CODE REVIEW

**Review by Pelentan — APPROVED** [review]

**Comment by pete-nerantzinis** [comment]
Just tested this in IE, i'm still getting the bug where the save button doesn't appear.

**Review by pete-nerantzinis — APPROVED** [review]
Works great!

File: LEAF_Request_Portal/admin/templates/mod_form.tpl [hunk 1]
```diff
@@ -970,7 +970,7 @@ function getForm(indicatorID, series) {
                 <div id="container_indicatorGrid" style="display: none"><span style="position: absolute; color: transparent" aria-atomic="true" aria-live="polite" id="tableStatus" role="status"></span>\
                 </br><button class="buttonNorm" onclick="addCells(\'column\')"><img src="../../libs/dynicons/?img=list-add.svg&w=16" style="height: 25px;"/>Add column</button>&nbsp;\
                 </br></br>Columns:<div border="1" style="overflow-x: scroll; max-width: 100%; border: 1px black;"></div></div>\
-                <div style="float: right">Default Answer<br /><textarea id="default"></textarea></div></fieldset>\
+                <fieldset><legend>Default Answer</legend><textarea id="default" style="width: 50%;"></textarea></fieldset></fieldset>\
```
[the remark this answers is not in the document]
Comment (pete-nerantzinis) [reply LEAF_Request_Portal/admin/templates/mod_form.tpl:973]: Its a fieldset within a fieldset, its closed from the previous fieldset thats why its their twice.

[the remark this answers is not in the document]
Comment (gsteinhoff) [reply LEAF_Request_Portal/admin/templates/mod_form.tpl:973]: The first ending fieldset is for the fieldset on the same line (973).  The second ending fieldset is connected to the one opened on line 498.  We just converted the Div we removed into a fieldset to make it look better in the field form input.

**Review by Pelentan — APPROVED** [review]

**Review by nfceek — APPROVED** [review]

**Pull request merged**